src/components/va-accordion/va-accordion.e2e.ts

```diff
@@ -103,6 +103,7 @@ describe('va-accordion', () => {
         header: "First item",
         subheader: "First subheader",
         level: 2,
+        sectionHeading: null,
```

Comment:
Can we also have a test showing that the prop gets mapped the the event correctly?

Reply:
Sure thing.